tests/suites/1_stateful/00_copy/00_0000_copy_from_ipfs_location.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+TABLE=ontime200
+FILE=ontime_200
+QMHASH=QmPpCt1aYGb9JWJRmXRUnmJtVgeFFTJGzWFYEEX7bo9zGJ
```

Comment:
Visit https://ipfs.filebase.io/ipfs/QmPpCt1aYGb9JWJRmXRUnmJtVgeFFTJGzWFYEEX7bo9zGJ for files list.

Don't add tests that we don't have related files.

Reply:
And use `https://ipfs.filebase.io` as endpoint which is faster than `https://ipfs.io` for our test cases.